spooky-doodle/Bangazon-Workforce-Management
# Edit employee

# Description

Users can now assign or unassign multiple computers at once for a specific employee

Fixes #8 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Create a computer without assigning it to an employee. Nav to employees. View Details, noting which computers they currently have. Edit the same employee, adding or removing computers. View employee details once again and check that the update was successful.


# Checklist:
